new PR

esselesse opened the pull request.

esselesse committed:
first try

esselesse committed:
little fixes

esselesse committed:
css's (something worked, so kinda working)

esselesse committed:
ОНО ЦВЕТНОЕ!

esselesse committed:
temp

esselesse committed:
temp

esselesse committed:
temp

esselesse committed:
temp

esselesse committed:
it seems working

esselesse committed:
it seems working

esselesse committed:
it seems working

esselesse commented:
по-другому не хочет пулл создавать

DmitryMakhnev commented on .idea/Viktor_Fridman.iml:
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
файлы идеи не надо коммитить

DmitryMakhnev commented on firstTodo/Todo_FIRST.html:
```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
```
не нужны папки относительно дз

DmitryMakhnev commented on firstTodo/Todo_FIRST.html:
```diff
@@ -0,0 +1,117 @@
+            Todo
+        </h3>
+
+        <h5 class="main-title3" align="right" aria-label="todo2"> <!--font color="#00ffff"-->
```
Зачем 3 заголовка?

DmitryMakhnev commented on firstTodo/Todo_FIRST.html:
```diff
@@ -0,0 +1,117 @@
+        <div class = "addAndSelect">
+            <div class="selector">
+                <button class="selectAll" aria-label="selectAllTodoItems">
+                    <img  alt="Select all" src="src/images/icons/select-all.png"/>
```
Картинка должна ставиться через css

esselesse replied:
так это ж первый самый вариант. без css и прочего.

посмотрите todo_last

DmitryMakhnev commented on firstTodo/Todo_FIRST.html:
```diff
@@ -0,0 +1,117 @@
+        </h5>
+    </div>
+
+    <div class="mainBody" role="main" >
```
Классы называем через `-`

DmitryMakhnev commented on firstTodo/Todo_FIRST.html:
```diff
@@ -0,0 +1,117 @@
+
+    <div class="mainBody" role="main" >
+        <div class = "addAndSelect">
+            <div class="selector">
```
Это элемент блока добавления а не отдельный блок

DmitryMakhnev reviewed (commented):
ДЗ очень похоже на дз https://github.com/ok-technopolis-autumn-2017/Egor_Gorbatov/pull/4/files
Вы можете почитать замечания и рекомендации там

esselesse committed:
refactor naming.

esselesse committed:
added EventListeners

esselesse committed:
cleared logging and refactored

esselesse committed:
жесть какая-то. нафиг эту 6ую версию, разбираемся с 5ой

esselesse committed:
начали движение в сторону капсулирования. версия НЕРАБОЧАЯ.
необходимо объединить функции по конструкторам; сделать для каждого конструктора - свой файл, свой прототип, свой экспортс.

esselesse committed:
scss's

esselesse committed:
scss's refactor

esselesse committed:
scss's refactor

esselesse committed:
scss's refactor

esselesse committed:
scss's refactor

esselesse committed:
scss's refactor

esselesse committed:
scss's refactor

esselesse committed:
scss's refactor

esselesse committed:
scss's & CSS refactor

esselesse committed:
reallocation and refactor

esselesse committed:
CHECKPOINT

esselesse committed:
refactor

esselesse committed:
remake css

esselesse committed:
refac

esselesse committed:
seems good

esselesse committed:
ref

esselesse committed:
ref scss

esselesse committed:
ref scss

esselesse committed:
ref.
осталось сформировать из всех js скриптов общий bundle.js и всё должно заработать.

esselesse committed:
webpacking

esselesse committed:
webpacking

esselesse committed:
webpacking

esselesse committed:
ref

stanislav3316 commented:
Да и тут ДЗ очень похоже на списанное, плохо

esselesse commented:
я сохраняю части разных дз и осознаю, как это всё работает)

esselesse committed:
переосмысление

esselesse committed:
structurising scss

esselesse committed:
structurising scss

esselesse committed:
structurising scss

esselesse committed:
structurising scss

esselesse committed:
structurising scss

esselesse committed:
structurising scss

esselesse committed:
CSS done

esselesse committed:
CSS done again

esselesse committed:
почему-то всё фейлится. нужно время, чтобы разобраться, а его совсем нет(

esselesse commented:
в общем, почти разобрался.
осталось деструктуризировать js и мигрировать его инъекцию из index'a в js-файл. + разобраться в наложениях шкурок классов друг на друга в css+html.

к сожалению, факт того, что изначально работа была сдана 28 декабря, а проверена лишь 30 января, когда я был в командировке (из которой и вернулся, собственно, вчера, что-то поковыряв в пути) несколько удручает. :(

clhost commented:
stanislav3316, согласен с Вами.

DmitryMakhnev reviewed (commented):
1) В макете так и не появилось 3 ёх хедеров
2) Если вы берёте части других дз, смотрите чтобы они были заапрувлены, т.к. иначе получается каша.
3) В текущем виде смотреть не вижу смысла. Пожалуйста подойдите к решению домашнего задания без набора копипаст или возьмите готовое решение преподавателя
4) идеевские файлы надо записывать в гит игнор

esselesse commented:
странно. у меня три разноцветных заголовка, как я и задумывал
..
в каком контексте - брать готовое решение преподавателя?

esselesse committed:
gitignored

esselesse committed:
вот теперь более-менее правильная разметка.

esselesse committed:
fix

esselesse committed:
fix
осталось починить фильтры.

esselesse committed:
css refactor

esselesse committed:
fixed select-all icon

esselesse committed:
fixed _hide filtration.
everything works.

в css'ках могут быть перепеременирование (или как там?..), могу вычистить, но сейчас башка уже не варит.

esselesse committed:
fix-refactor
